docs/fundamentals/code-analysis/quality-rules/ca1844.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "CA1844: Provide memory-based overrides of async methods when subclassing 'Stream' (code analysis)"
+description: "Learn about code analysis rule CA1844: Provide memory-based overrides of async methods when subclassing 'Stream'"
+ms.date: 05/18/21
```

Comment:
```suggestion
ms.date: 05/18/2021
```

Reply:
@Youssef1313 you are a lifesaver. I was tearing my hair out trying to figure out why it wouldn't build.

Reply:
It will probably still fail though 😄 

Other violations are about xrefs not found.

Reply:
Where do I actually see the build warnings? I wasn't able to find them, it just said there were build warnings.

Reply:
@NewellClark You can see them under the details of `OpenPublishing.Build`. Unfortunately, there are recently a lot of warnings in existing files. So you would need to search for the file name you're adding in the table.

![image](https://user-images.githubusercontent.com/31348972/118760679-82320c80-b873-11eb-9657-9c8c01690a96.png)